CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
-## [Unreleased]
+
+[//]: # (## [Unreleased])
```

Comment:
FYI this works too:

```markdown
<!-- ## [Unreleased] -->
```

Reply:
Good to know, the syntax I used is pretty weird 😬